src/callbacks/onResize.js

```diff
@@ -82,25 +82,39 @@ export default function onResize() {
 
     //Offset bottom x-axis to prevent overlap with final ID.
     const bottomXaxis = this.svg.select('.x.axis'),
-        bottomXaxisTitle = bottomXaxis.select('.axis-title');
-    bottomXaxis.attr(
-        'transform',
-        `translate(0,${+bottomXaxis
-            .attr('transform')
-            .split(',')[1]
-            .split(')')[0] + this.y.rangeBand()})`
-    );
+        bottomXaxisTransform = bottomXaxis.attr('transform'),
```

Comment:
Not really sure what all this action is about - add a comment?

Reply:
I'm dropping the bottom x-axis by a rangeBand so that the final ID doesn't draw over the x-axis.